int32 number_of_tries
---
string results
